From 8cb4f80ef648b6aec9b0c1517cde9bfdfeb66b13 Mon Sep 17 00:00:00 2001
From: Fugang Duan <b38611@freescale.com>
Date: Sun, 4 Jan 2015 13:38:26 +0800
Subject: [PATCH] MLK-10072 ARM: dts: imx6sx: add uart5 dte pad set for
 imx6sx-sabreauto board

Add imx6sx-sdb baord uart5 DTE pad set. To avoid a flood of dts files,
there only comment out DTE pinctrl set. If user want to test DTE mode,
it needs to rebuild the DTB file.

(Cherry-picked and merged from commit 4bd6e1654495e190e61a70c9b2c44fda931e2727)

Signed-off-by: Fugang Duan <B38611@freescale.com>
---
 arch/arm/boot/dts/imx6sx-sabreauto.dts | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx-sabreauto.dts b/arch/arm/boot/dts/imx6sx-sabreauto.dts
index be1cb26..7661562 100644
--- a/arch/arm/boot/dts/imx6sx-sabreauto.dts
+++ b/arch/arm/boot/dts/imx6sx-sabreauto.dts
@@ -368,6 +368,15 @@
 			>;
 		};
 
+		pinctrl_uart5dte_1: uart5dtegrp-1 {
+			fsl,pins = <
+				MX6SX_PAD_KEY_ROW3__UART5_TX	0x1b0b1
+				MX6SX_PAD_KEY_COL3__UART5_RX	0x1b0b1
+				MX6SX_PAD_KEY_ROW2__UART5_RTS_B	0x1b0b1
+				MX6SX_PAD_KEY_COL2__UART5_CTS_B	0x1b0b1
+			>;
+		};
+
 		pinctrl_usbotg1_1: usbotg1grp-1 {
 			fsl,pins = <
 				MX6SX_PAD_GPIO1_IO10__ANATOP_OTG1_ID 0x17059
-- 
1.8.0

